### ISSUE TITLE
Fixed saveParsedFilesInfoToCache() in SassHandler.php

### ISSUE THREAD

**Issue by oneakroller** [artem-frolov/yii-sass#5]
```
/**
 * Save a list of parsed files to the cache
 * with the time files were last modified.
 * Must be called right after the compilation.
 *
 * @param string $sourcePath Path to the source SCSS file
 */
protected function saveParsedFilesInfoToCache($sourcePath)
{
    $parsedFiles                = $this->compiler->getParsedFiles();
    $parsedFiles[$sourcePath]   = filemtime($sourcePath);
    $parsedFilesWithTime        = array();

    foreach ($parsedFiles as $file=>$value) {
        $parsedFilesWithTime[$file] = filemtime($file);
    }
```

**Comment by artem-frolov** [comment]
@oneakroller What problem is this fix for? Putting time into the `$parsedFiles` where paths are stored, doesn't seem to be correct.
Thanks!

**Comment by aloumpas** [comment]
Hello, are you willing to fix the above bug?

**Comment by artem-frolov** [comment]
@aloumpas I still don't have a bug explanation. Please feel free to explain what is the problem and submit a pull request. Thank you!

**Comment by artem-frolov** [comment]
Fixed in #7
